locales/en.json

```diff
@@ -42,11 +42,12 @@
 	"driver_deleted": "driver deleted successfully",
 	"zero_users": "No users in the database",
 	"address_required": "Adrress is required",
-	"operator_deleted": "operator_deleted",
+	"operator_deleted": "Operator deleted successfully",
 	"driver_updated": "Driver updated successfully",
 	"national_ID_exist": "User with provided national ID exists",
 	"license_ID_exist": "User with provided license exists",
 	"not_Found": "not_Found",
```

Comment:
Good job Mr. Yvad, if you are working on feedback from others, remember to change this, looks like the key and value are the same... on line 45 and 49

Reply:
yeahhh! thanks